xen.spec.in

```diff
@@ -118,7 +114,7 @@ Patch706: patch-mini-os-link-to-libgcc.a-to-fix-build-with-gcc7.patch
 # GCC8 fixes
 Patch714: patch-tools-kdd-mute-spurious-gcc-warning.patch
 # GCC9 fixes
-Patch716: patch-python-fix-Wsign-compare-warnings.patch
+#Patch716: patch-python-fix-Wsign-compare-warnings.patch
```

Comment:
@marmarek: I had to comment this patch when using the Fedora python3 changes

Reply:
I think it's part of the big python3 patch already. In that case, you can simply remove it.

Reply:
Ok I do that.